cli/src/lib.rs

```diff
@@ -149,23 +151,29 @@ impl NodeBuilder {
         self
     }
 
+    /// Adds a service instance that will be available immediately after creating a genesis block.
+    ///
+    /// Make sure that artifact for this instance is provided to this builder.
```

Comment:
Is this comment outdated? As far as I see, the artifact is added automatically in the `genesis_config` method. Or does the comment refer to the Rust service factory? In this case, I'd rephrase as

> For Rust services, the service factory needs to be supplied via `with_rust_service` separately.